src/main/resources/migrations/V7__create_table_voting_stats.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Copyright (C) 2019 Kaleidos Open Source SL
+--
+-- This file is part of Don't Worry Be Happy (DWBH).
+-- DWBH is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- DWBH is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with DWBH.  If not, see <https://www.gnu.org/licenses/>
+--
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS voting_stats (
+  id UUID NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
We don't initialize by default ids in database configuration, it's something hibernate takes care of. Specially because it requires the database to have install a specific extension.

![extension_required](https://user-images.githubusercontent.com/1414131/85390181-bb210e80-b548-11ea-9012-ff65e9291385.png)

Reply:
We have come to the following conclusion: remove the default uuid in the table, but provide it with a value in the insert query